run-tests-with-build-optaplanner-quickstarts-parent/optaplanner-quickstarts-project-sources-test/pom.xml

```diff
@@ -51,23 +59,78 @@
         <artifactId>maven-invoker-plugin</artifactId>
         <configuration>
           <parallelThreads>1</parallelThreads>
-          <!--
-          script deciding if included project is:
-            executed (either script missing or returning true)
-              or
-            skipped (when returns false or throws error) -->
-          <selectorScript>${maven.modules.resolution.selector.script.name}</selectorScript>
-          <pomIncludes>
-            <!-- includes everything, filtering is done in resolve-includes.groovy script
-            which adds invoker-run.groovy script containing 'return false' near each pom.xml
-            that should be excluded.-->
-            <pomInclude>**/pom.xml</pomInclude>
-          </pomIncludes>
-          <pomExcludes>
-            <!-- Excludes work still as expected. -->
-          </pomExcludes>
         </configuration>
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>non-native</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-invoker-plugin</artifactId>
+              <configuration>
+                <properties combine.children="append">
+                  <skipTests>true</skipTests>
+                </properties>
+                <!--
+                script deciding if included project is:
+                  executed (either script missing or returning true)
+                    or
+                  skipped (when returns false or throws error) -->
+                <selectorScript>${maven.modules.resolution.selector.script.name}</selectorScript>
+                <pomIncludes>
+                  <include>${invoker.pom.include}</include>
+                </pomIncludes>
+                <pomExcludes>
+                  <!-- Excludes work still as expected. -->
+                </pomExcludes>
+              </configuration>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+      </build>
+    </profile>
+    <profile>
+      <id>native</id>
+      <activation>
+        <property>
+          <name>native</name>
+        </property>
+      </activation>
+      <properties>
+        <!-- native builds make sense just for quarkus examples -->
```

Comment:
This is a leftover from kogito-examples. Here you're not using any property like `invoker.pom.include`, if so remove the properties section altogether.

Reply:
Fixed 69c1c08c